src/coreComponents/physicsSolvers/wavePropagation/WaveSolverBase.cpp

```diff
@@ -68,6 +78,15 @@ WaveSolverBase::~WaveSolverBase()
 }
 
 
+void WaveSolverBase::reinit()
+{
+  DomainPartition & domain = this->getGroupByPath< DomainPartition >( "/Problem/domain" );
```

Comment:
`getGlobalState().getProblemManger().getDomainPartition()`

Reply:
Done.

Reply:
If I'm not mistaking, I think I can't do this because the `getGlobalState()` method is private within `GeosxState.hpp`.

Reply:
Nope, `getGlobalState()` is public.

Reply:
Right I just forgot to include the `GeosxState.hpp` file. Done